### DISCUSSION TITLE
Should packer_compiled.lua be checked into version control ?

### DISCUSSION THREAD

**Discussion by nicos68** [wbthomason/packer.nvim#1126] [Q&A]
I've checked on some well-known nvim/nvim plugin developers and they use packer but do not push their `packer_compile.lua` file to their repos. I even watched a video where the author says it should be put to `.gitignore`. But I've seen that the main devleoper of packer has it in his dotfiles repo, and it seems reasonable to me, as it looks like some sort of `package-lock.json` for packer.

What is the intent/view of the authors of packer about this detail ? Is there so good practice or does it depend ?

**Comment by rootix** [comment]
Not sure if you ever got your answer somewhere else, but i searched for this same question and found #462 which makes me ignoring this file in git.
